tools/rna_tools/aresite2/aresite2.xml

```diff
@@ -0,0 +1,53 @@
+<tool id="AREsite2_REST" name="Get motifs from AREsite2" version="0.1.2">
+    <description>AREsite2 REST Interface</description>
+    <requirements>
+        <requirement type="package" version="2.12.4">requests</requirement>
+        <requirement type="package" version="1.10.0">six</requirement>
+    </requirements>
+    <command>
+<![CDATA[
+python $__tool_directory__/aresite2.py
+-g $gene
+-m "$motif"
```

Comment:
single quotes ... reason is that double quotes are evaluated in bash and can cause harm